src/pages/guides/api/eventsingress_api.md

```diff
@@ -63,3 +65,18 @@ The API returns
 * HTTP Status 204 (No Content) if there are no registrations for the event,
 * HTTP Status 429 (Too Many Requests) if your api-key is being throttled (see our [Throttling Policy](#throttling-policy)).
 * as well as the usual (4xx/5xx) error codes if there was an issue in processing the request.
+
+## HIPAA Compliance Support
+
+I/O Events is compliant with [HIPAA regulation](https://www.hhs.gov/hipaa/for-professionals/privacy/index.html#:~:text=The%20HIPAA%20Privacy%20Rule%20establishes,care%20providers%20that%20conduct%20certain).
+The event producers are responsible to signal if the event contains PHI data by setting the optional `x-event-phidata` header to `true` in the request.
+When the header is present and set to `true`:
+* the resulting CloudEvents-formatted event which is produced will be enriched with a [CloudEvents extension](https://github.com/cloudevents/spec/blob/main/cloudevents/primer.md#cloudevent-extension-attributes), namely, a `phidata` node, set to `true`,
+* with the above `phidata` extension receivers will be able to identify which events contain PHI data and handle them accordingly,
+* audit trace logs will be preserved for 1 year in a HIPAA-compliant storage,
+* the `id` field in the CloudEvents payload will be used as a `trace-id` for the above audit trace logs.
+
+You can store the `id` in your own logs or storage solution for traceability, this will give you the 
+possibility to later ask I/O Events support to retrieve the audit trace logs for a given set of `ids`
+you produced, and by storing what `ids` correspond to what PHI data you handle, you will be able to
+easily provide evidences of how the patient data was handled in case of an audit.
```

Comment:
Shall we repurpose this information to make sure the customer understands the implications of not including this header? We could also capture it in a separate section.

Reply:
Thanks, yes reworked it :)